RetrospectiveExtension.Frontend/components/feedbackItem.tsx

```diff
@@ -831,131 +831,125 @@ class FeedbackItem extends React.Component<IFeedbackItemProps, IFeedbackItemStat
             </div>
           </DocumentCard>
         </div>
-        {!this.state.isDeleteItemConfirmationDialogHidden &&
```

Comment:
it looks like a lot of this file was changed - however all I did was move the ...Hidden states into the `<Dialog>` to set the hidden value. 

For example:

```
 {!this.state.isDeleteItemConfirmationDialogHidden &&
  <Dialog
     hidden={false}>...
  </Dialog>
}
```
became: 
```
<Dialog
   hidden={this.state.isDeleteItemConfirmationDialogHidden }>...
</Dialog>
```

Reply:
Hey, I really appreciate these kinds of comment. Really help reviewers to check the code faster.